librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -429,6 +429,118 @@ commands:
       - name: type
         type: RZ_CMD_ARG_TYPE_NUM
         optional: true
+  - name: px
+    summary: Show hexdump
+    subcommands:
+      - name: px
+        summary: show hexdump
+        cname: print_hexdump
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pxd
+        summary: show signed integer dump
+        cname: print_hexdump_signed_int
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: n
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
```

Comment:
this is an option... another option is to mimic the behaviour we have right now, with subcommands like `pxd1`,`pxd2`, `pxd4`, etc. Like what is done for `wv?` for example.

Reply:
Actually... I wonder whether it would make sense to follow the same pattern as the regular `px` command... That is: `pxd`, `pxdh`, `pxdw`, `pxdq`. WDYT @XVilka @wargio ?

Reply:
Yes, that would make more sense and be more consistent.

Reply:
yup